rpm_spec/manageiq.spec.in

```diff
@@ -34,6 +34,7 @@ Source3:  %{name}-manifest-%{version}.tar.gz
 %{product_summary}
 
 %prep
+umask 0007
```

Comment:
Should this be using `defattr`?

Reply:
This sets the umask at build time, so all the `mkdir -p` commands and stuff will get the correct `chmod`.

We will no longer depend upon the `%file` section to fix/change the attributes because they will have the correct values in the first place.